src/java/org/apache/cassandra/service/reads/ReplicaFilteringProtection.java

```diff
@@ -197,6 +198,9 @@ public UnfilteredRowIterators.MergeListener getRowMergeListener(DecoratedKey par
                 for (int i = 0; i < sources.size(); i++)
                     builders.add(i, new PartitionBuilder(partitionKey, sources.get(i), columns, stats));
 
+                boolean[] silentRowAt = new boolean[builders.size()];
+                boolean[] silentColumnAt = new boolean[builders.size()];
```

Comment:
The class JavaDoc talks about silent rows (`the rows in a replica response that don't have a corresponding row in other replica responses`). We should probably update that also to mention the new silent columns.

Reply:
Agree. Will add that.